### DISCUSSION TITLE
zh/docs/syncer/WeCom/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2864] [Docs comments]
# zh/docs/syncer/WeCom/

使用WeCom Syncer同步数据库

https://casdoor.org/zh/docs/syncer/WeCom/

**Comment by Deali-Axy** [comment]
为啥添加的时候只有 database 和 keycloak ，没有 wecom 类型的同步器？

**Reply by yhgroup** [reply]
确实没有 wecom 类型的同步器！密码登录可强制MFA，但企业微信登录强制MFA失效的，准备企业微信登录自动创建账号了并启用MFA了。

**Reply by Deali-Axy** [reply]
好的。

**Comment by regend** [comment]
企业微信的同步器下线了吗？

**Comment by ashtonli** [comment]
现在的版本是没有wecom类型的同步器了吗？还是说在新的版本里面？
